src/fodesystem/PIPECE.jl

```diff
@@ -151,12 +151,12 @@ function solve(prob::FODESystem, h, ::PECE)
     end
 
     t = t[1:N+1]; y = y[:, 1:N+1]
-    return t, y
+    return FODESolution(t,y)
```

Comment:
The `FODESolution` is now only support single term FODE and Multi terms FODE, the solution types for `FODESystem` has not been implemented so far.
```suggestion
    return t, y
```